docs/source/recipes/color-picker-widget.md

```diff
@@ -0,0 +1,161 @@
+---
+myst:
+  html_meta:
+    "description": "How to use the color picker widget in blocks settings and forms"
+    "property=og:description": "How to use the color picker widget in blocks settings and forms"
+    "property=og:title": "How to use the color picker widget"
+    "keywords": "Volto, Plone, frontend, React, blocks, forms, widget, color, picker"
+---
+
+# Color picker widget
+
+Volto comes with a color picker widget that can be used in any Volto form.
```

Comment:
People might be expecting a color widget that allows picking any color, so we might want to clarify that this one picks from a preset list of colors.

Reply:
I improved it in the next sentence.